R/histogram-datamapping.R

```diff
@@ -6,6 +6,8 @@ HistogramDataMapping <- R6::R6Class(
   "HistogramDataMapping",
   inherit = XYGDataMapping,
   public = list(
+    #' @field frequency logical defining if histogram displays a frequency in y axis
+    frequency = NULL,
```

Comment:
shouldn't it be `frequency = FALSE,` instead of NULL?

Reply:
Here, both work as the actual value is defined during the initialization by `frequency=FALSE`.
Unless `unlock_object=TRUE`, a field named `frequency` has to be available before the initialization of the object no matter its value.